AlicerceTests/Logging/ConsoleLogDestinationTests.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  ConsoleLogDestinationTests.swift
+//  Alicerce
+//
+//  Created by Meik Schutz on 04/04/17.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import XCTest
+@testable import Alicerce
+
+class ConsoleLogDestinationsTests: XCTestCase {
+
+    fileprivate let log = Log()
+    fileprivate let queue = Log.Queue(label: "ConsoleLogDestinationsTests")
+    fileprivate let expectationTimeout: TimeInterval = 5
+    fileprivate let expectationHandler: XCWaitCompletionHandler = { error in
+        if let error = error {
+            XCTFail("🔥: Test expectation wait timed out: \(error)")
+        }
+    }
+
+    override func tearDown() {
+        super.tearDown()
+        log.removeAllDestinations()
+    }
+
+    func testConsoleLogDestination() {
+
+        // preparation of the test subject
+
+        let destination = Log.ConsoleLogDestination(minLevel: .verbose,
```

Comment:
To properly test this we can change this Destination to receive a `print` function as a dependency. 😉